Base/usr/share/man/man7/setuid_overview.md

```diff
@@ -0,0 +1,71 @@
+## Name
+
+Overview of real, effective, and saved user and group ids
+
+## Description
+
+Each process runs as user and a group. By default, a process started by a user has its user id set to that user and its group id set to that user's primary group.
```

Comment:
That's, uh, kinda true, but not really. Users don't start processes; processes spawn other processes — perhaps, on behalf of a user. As you're surely know, the IDs a process runs at are determined/affected by ~three~ four simple rules:
* The kernel always starts processes as UID/GID 0. In particular, PID 4, the first userspace process, is spawned as root.
* A process can change its own IDs by one of these `set*()` calls.
* IDs are inherited on fork.
* IDs can be changed at exec time if the filesystem feels like it.

So these rules have to be clearly written out on this page. Now if, say, anon runs a shell with UID 100 and it spawns some new process, it inherits UID 100 by default — so in that sense, yes, the programs started by a user run as their UID/GID. But it's just that, a common case, not the underlying mechanism.